app/views/my_modules/overview.html.erb

```diff
@@ -0,0 +1,19 @@
+<% provide(:head_title, raw(t('my_modules.overview.head_title', project:
+@project.name, module: @my_module.name))) %>
+<%= render partial: 'shared/sidebar' %>
+<%= render partial: 'shared/secondary_navigation' %>
+<%= render partial: 'module_header' %>
+
+<div id='widget-container'>
+  <% if @my_module.widgets.count == 0 %>
+  <div data-role="no-widgets-text">
+    <em><%= t("my_modules.overview.widgets.none") %></em>
+  </div>
+  <% else %>
+  <% @my_module.widgets.order(:position).each do |widget| %>
+  <%= render partial: 'widgets/widget.html.erb', locals: { widget: widget } %>
+  <% end %>
+  <% end %>
```

Comment:
I'm actually using the _atom-beautify_, as specified in the wiki codestyle guide. It formats the code so that HTML elements are indented correctly, but Ruby code stays on the same indentation level as the previous HTML tag. To make application (_*.html.erb_) code consistent, we should probably do it this way. We should also probably make a pull request which formats all of the _*.html.erb_ code with the _atom-beautify_, as it can be done in a jiffy.